### ISSUE TITLE
Broadcasting error after reshape

### ISSUE THREAD

**Issue by spenrich** [cvxgrp/cvxpy#872]
With both variables and parameters, I get an unexpected ValueError depending on whether I reshape with a tuple or a list.
```
import cvxpy as cvx
n = 6
A = cvx.Variable([n, n])
b = cvx.Variable(n**2)

# This does not work
B = cvx.reshape(b, [n, n])
print(A + B)  #<- throws  "ValueError: Cannot broadcast dimensions  (6, 6) [6, 6]"

# This works
B = cvx.reshape(b, (n, n))   #<- reshaped with tuple instead
print(A + B)
```

**Version**
 - OS: MacOS 10.14.6 (Mojave)
 - CVXPY Version: 1.1.0a1

**Comment by akshayka** [comment]
@spenrich , thanks for reporting this. We'll merge the fix (PR #874) into master soon, and we'll push a new version to pypi within a few days.

**Comment by rileyjmurray** [comment]
@akshayka do you want to close this, in view of release https://github.com/cvxgrp/cvxpy/releases/tag/v1.1.0a2 ?
